api/tests/src/search/backend/test_load_opportunities_to_index.py

```diff
@@ -1,12 +1,25 @@
+from unittest.mock import Mock
+
 import pytest
 
+from src.db.models.opportunity_models import OpportunitySearchIndexQueue
 from src.search.backend.load_opportunities_to_index import (
     LoadOpportunitiesToIndex,
     LoadOpportunitiesToIndexConfig,
 )
 from src.util.datetime_util import get_now_us_eastern_datetime
 from tests.conftest import BaseTestClass
-from tests.src.db.models.factories import OpportunityFactory
+from tests.src.db.models.factories import OpportunityFactory, OpportunitySearchIndexQueueFactory
+
+
+@pytest.fixture
+def mock_search_client():
```

Comment:
I don't think you use this anywhere

Reply:
Good catch - was part of a test I wrote trying out a mock service